workflow/deploy.md

```diff
@@ -51,16 +51,12 @@ In order to deploy to Kubernetes you will need to create 2 Jenkins jobs on the w
 
 Update the **apply** job:
 
-- Go to the [configure page](https://jenkins.canonical.com/webteam/job/apply-production-deployment-configs/configure)
-- Add these lines:
+- If you are going to deploy a new site, check that the file `/sites/<DNS_OF_THE_WEBSITE>.yaml` exists in the [deployment configs repository](https://github.com/canonical-web-and-design/deployment-configs)
+- Go to the [staging configuration](https://jenkins.canonical.com/webteam/job/apply-staging-deployment-configs/configure) and [production configuration](https://jenkins.canonical.com/webteam/job/apply-production-deployment-configs/configure) on Jenkins.
+- Add this line at the end of both build scripts:
 
 ```bash
```

Comment:
This code block should be indented by 2 spaces or so so it forms part of the previous list item